src/foam/u2/layout/Layouts.js

```diff
@@ -1,23 +1,207 @@
 foam.CLASS({
   package: 'foam.u2.layout',
-  name: 'RowLayout',
+  name: 'Rows',
   extends: 'foam.u2.Element',
+
+  requires: [
+    'foam.u2.layout.Row',
+  ],
+
+  documentation: `
+    A layout for a group of rows layed out vertically.
+  `,
+
   methods: [
     function add() {
-      console.log('Adding row!');
-      return this.SUPER.apply(this, arguments);
+      /**
+       * ! Trying to add a border like with Col breaks the row, we can look into later
+       * ! As card rows aren't probable right now
```

Comment:
I think rows of cards will be the more common use case to start because the sectioned detail view will be rows of cards.